### ISSUE TITLE
DockerId vs. DockerID

### ISSUE THREAD

**Issue by iann0036** [aws/amazon-ecs-agent#1497]
### Summary
DockerId and DockerID are used interchangeably depending upon task networking mode of the task

### Description
Tasks in host networking mode have their containers produce "DockerID" in the metadata service whereas tasks in awsvpc mode produce "DockerId"

### Expected Behavior
The same key for both types of tasks (don't care which one)

### Observed Behavior
Different keys

### Environment Details
N/A

### Supporting Log Snippets
```
$ curl -s http://localhost:51678/v1/tasks?taskarn=...
{...,"Containers":[{"DockerId":"xxx",}]}

$ curl -s http://localhost:51678/v1/tasks?taskarn=...
{...,"Containers":[{"DockerID":"yyy",...}]}
```

**Comment by iann0036** [comment]
It does appear that the presence of either one or many PortMappings also affects this behaviour.

**Comment by iann0036** [comment]
It appears this is not actually based on task networking mode and is arbitrary.

**Comment by iann0036** [comment]
Caused by https://github.com/aws/amazon-ecs-agent/pull/1437/commits/5970f46f68f332a8d8cbc5435d8b0c7f18c0e48f and fixed in 1.19.1
